popout3d_metainfo.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You need to reference this file in your manifest and install it at `/app/share/metainfo/${FLATPAK_ID}.metainfo.xml`.

Reply:
Also your metainfo file should include [OARS](https://hughsie.github.io/oars/) and `<releases>` section.

Reply:
Reference - todo. Install it - done.
OARS - done, <releases> done.